### PR TITLE
Add amsmath support

### LINKED ISSUE #1057: Add amsmath support
## Description
I'd like amsmath to be included in the LaTeX command for Sir Lancebot.

## Reasoning
Amsmath is a widely used package for things such as including text in equations or aligning multiple equations. Its absence has long been a thorn in my side, and an extremely annoying one at times.

## Proposed Implementation
The amsmath package should be included in the default template, so it gets imported for every LaTeX command run.
This can be done by including the following text on a new line after the first line:
`\usepackage{amsmath}`

## Would you like to implement this yourself?

- [x] I'd like to implement this feature myself
- [ ] Anyone can implement this feature

### PR DESCRIPTION
**Pull request by PH-KDX**
## Relevant Issues

Closes #1057 

## Description
This PR includes the amsmath in the default LaTeX template using `\usepackage{amsmath}` so that people can use Amsmath commands in the .latex command.

## Did you:

- [x] Join the [**Python Discord Community**](https://discord.gg/python)?
- [x] Read all the comments in this template?
- [x] Ensure there is an issue open, or link relevant discord discussions?
- [x] Read and agree to the [contributing guidelines](https://pythondiscord.com/pages/contributing/contributing-guidelines/)?

### CODE REVIEW

**Review by wookie184 — APPROVED** [review]
Seems fine

File: bot/resources/fun/latex_template.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 \documentclass{article}
+\usepackage{amsmath}
```
Comment (ShakyaMajumdar) [inline bot/resources/fun/latex_template.txt:2]: https://github.com/python-discord/sir-lancebot/issues/1057#issuecomment-1131173640
```suggestion
\usepackage{amsmath, amssymb}
```

Comment (PH-KDX) [reply bot/resources/fun/latex_template.txt:2]: Not sure about this one; I can't remember the last time I or anyone else needed to use amssymb in PyDis.
Although on the other hand, it doesn't really add much compile time, so whatever.

Comment (Xithrius) [reply bot/resources/fun/latex_template.txt:2]: Probably best to add it just in case, you never know.

Comment (decorator-factory) [reply bot/resources/fun/latex_template.txt:2]: +1 for `amssymb`, it has a bunch of useful stuff like `\mathbb` for bold letters and a billion different arrows

**Review by decorator-factory — APPROVED** [review]

File: bot/resources/fun/latex_template.txt (hunk 1 above)
Comment (PH-KDX) [reply bot/resources/fun/latex_template.txt:2]: Alright, I'll add it now.

**Review by ShakyaMajumdar — APPROVED** [review]

**Pull request merged**